api/src/routing.js

```diff
@@ -183,22 +182,12 @@ app.get('/dbinfo', (req, res) => {
 
 app.get(appPrefix, (req, res) => proxy.web(req, res));
```

Comment:
Less important, but routes which include `appPrefix` in their path will not pick up the "medic dbname-alias" that caused this regression.  Like this line on 184 and 2 others [here](https://github.com/medic/medic/blob/f053eba58087f3c3afab84e5c18c02d547a0f917/api/src/routing.js#L398). 

Should we be using the "medic dbname-alias" for these routes also? Can file new issue or possibly handle here?

Reply:
I saw that you were updating this very line in a PR, is my memory failing me?

Reply:
ah, it just got merged. https://github.com/medic/medic/commit/17499520f42fdd732877eba9f0c25aa9ecf493a0#diff-56e3c7d50c6dcffca3c40b498f2f568aR184

Reply:
I'll try to handle all these within this issue, ideally.